{"archive_info": {"hash": "sha256=3111b9d131c238bec2f8f516e123e14ba243563fb135d3fe885990585aa7795b"}, "url": "https://files.pythonhosted.org/packages/02/cc/b7e31358aac6ed1ef2bb790a9746ac2c69bcb3c8588b41616914eb106eaf/exceptiongroup-1.2.2-py3-none-any.whl"}